### DISCUSSION TITLE
Scene Metadata: Save time from which the video playback should begin

### DISCUSSION THREAD

**Discussion by lamyergeier** [stashapp/stash#5050] [Q&A]
Most videos have some loud music in the beginning when they show the logo of the studio, it can be annoying and startling to hear that, can I save for each video a time stamp from which every play should begin? 

For example for a particular video I want the playback to always begin from 1 minute 37 seconds.

**Comment by DogmaDragon** [comment]
Not possible natively, but there is a plugin that adds that functionality. 
https://github.com/feederbox826/plugins/tree/main/plugins/skip-intro
https://github.com/feederbox826/stash-skip-intro

**Comment by lamyergeier** [comment]
@DogmaDragon I installed the plugin, how to define the start time?

**Reply by DogmaDragon** [reply]
It uses the database from [feederbox826/stash-skip-intro](https://github.com/feederbox826/stash-skip-intro). You can create your own or submit it to that repository.
